server/routerlicious/packages/services/src/throttler.ts

```diff
@@ -61,14 +61,13 @@ export class Throttler implements IThrottler {
                     eventName: "throttling",
                 },
             });
-            Lumberjack.info(`Throttled: ${id}`, {
-                messageMetaData: {
-                    key: id,
-                    reason: cachedThrottlerResponse.throttleReason,
-                    retryAfterInSeconds,
-                    eventName: "throttling",
-                },
-            });
+            Lumberjack.info(
+                `Throttled: ${id}`,                         {
+                {
+                    [CommonProperties.telemetryGroupName]: "throttling",
```

Comment:
Any reason why properties `reason` and `retryAfterInSeconds` are missing in Lumebrjack, given they were used in logger above?

Reply:
Good call, updated :)